server/src/modules/stats/board/board.stats.module.ts

```diff
@@ -15,9 +17,10 @@ import {
         schema: BoardStatsSchema,
       },
     ]),
+    UsersModule,
   ],
   controllers: [BoardStatsController],
-  providers: [BoardStatsService],
+  providers: [BoardStatsService, UsersService],
```

Comment:
UsersService should not be a provider of this module. Currently it is a provider of UsersModule and it cannot serve as a second provider